apps/src/code-studio/components/progress/teacherPanel/TeacherPanel.jsx

```diff
@@ -38,7 +38,7 @@ class TeacherPanel extends React.Component {
       id: PropTypes.number.isRequired,
       name: PropTypes.string.isRequired
     }),
-    unitHasLockableLessons: PropTypes.bool.isRequired,
+    unitHasLockableLessons: PropTypes.bool,
```

Comment:
This was causing a warning in the console so I just updated while I was in here.

Reply:
Oops, just noticed that I had noticed the same issue but fixed it in a different way in https://github.com/code-dot-org/code-dot-org/pull/42791/files.  @maureensturgeon, do you have a preference for which way to fix this?

Reply:
(Dani, If the PR is otherwise ready, feel free to merge and I can resolve as needed in my PR.)